terraform/00-backend/.gitignore

```diff
@@ -0,0 +1,20 @@
+# Local .terraform directories
```

Comment:
One file per git project. You has two files in each directory with TF state

Reply:
Resolved